pallets/messages/src/lib.rs

```diff
@@ -179,16 +184,13 @@ pub mod pallet {
 			schema_id: SchemaId,
 			/// The block number for these messages
 			block_number: T::BlockNumber,
-			/// Number of messages in this block for this schema
-			count: u16,
 		},
 	}
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
-		fn on_initialize(current: T::BlockNumber) -> Weight {
-			let prev_block = current - T::BlockNumber::one();
-			Self::move_messages_into_final_storage(prev_block)
+		fn on_initialize(_current: T::BlockNumber) -> Weight {
```

Comment:
Keeping this around for potential retention policy implementation but open to remove it in case of objections

Reply:
I'm up for removing it. I think we might end up using the scheduler or something different, so best to clear it out.